checkov/common/output/record.py

```diff
@@ -89,7 +89,7 @@ def _code_line_string(code_block):
 
     def to_string(self, compact=False, use_bc_ids=False):
         status = ''
-        evaluation_message = f''
```

Comment:
is this breaking something?

Reply:
nope, the `f` before the string is not needed, because there is nothing to format 🙂